tests/numpy/assignment_test.py

```diff
@@ -36,6 +36,78 @@ def multiassign(B: dace.float64[1],
     assert B[0] == expected and C[1] == expected
 
 
+@compare_numpy_output(positive=True)
```

Comment:
We really need to add tests with symbolic sizes to explore the limits of what we can handle.